G/GR/build_tarballs.jl

```diff
@@ -18,13 +20,21 @@ fi
 
 if [[ $target == *"mingw"* ]]; then
     winflags=-DCMAKE_C_FLAGS="-D_WIN32_WINNT=0x0f00"
+    tifflags=""
```

Comment:
```suggestion
    tifflags=""
    update_configure_scripts
```

Reply:
Oh, the failure is in building a third-party library (ZeroMQ).  Why are we doing that since we have it already?

Reply:
GR requires an older ZeroMQ version than what is in the JLL.

Reply:
@giordano Should we merge the commit suggestion above?

We could build an older ZeroMQ as a statically built BuildDependency, so as to not build it in here. But that can also be done in a later build.

Reply:
> Should we merge the commit suggestion above?

That doesn't help because it's an external library that is failing to build.  I don't think there is any simple hook into that build process which allows us to add the `update_configure_scripts` command